troubleshooting/index.md

```diff
@@ -10,3 +10,124 @@ type: markdown
 
 Oh no! You have some problems? Let us help.
 
+## Envoy won't connect to my HTTP/1.0 service
+
+Envoy requires HTTP/1.1 or HTTP/2 traffic for upstream services. For example, when using [NGINX](https://www.nginx.com/) for serving traffic behind Envoy, you will need to set the [proxy_http_version](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_http_version) directive in your NGINX config to be "1.1", since the NGINX default is 1.0
+
+Example config:	
+
+```
+upstream http_backend {
+    server 127.0.0.1:8080;
+
+    keepalive 16;
+}
+
+server {
+    ...
+
+    location /http/ {
+        proxy_pass http://http_backend;
+        proxy_http_version 1.1;
+        proxy_set_header Connection "";
+        ...
+    }
+}
+```
+
+## Running multiple replicas of Mixer does not work as expected 
+
+For the alpha release, Mixer **must** be configured to run as a single instance within a cluster. We are working on improvements at the protocol, configuration, and deployment levels to support multiple instance and high-availability deployments. We expect to remove this limitation shortly after the initial alpha release.
```

Comment:
Use the version identifier instead of Alpha through the doc. It's {{stie.data.istio.version}}